sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -485,6 +490,14 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_adl_machines[] = {
 		.drv_name = "adl_cs35l41",
 		.sof_tplg_filename = "sof-adl-cs35l41.tplg",
 	},
+	{
+		.comp_ids = &essx_83x6,
+		.drv_name = "sof-essx8336",
+		.sof_tplg_filename = "sof-adl-es83x6", /* the tplg suffix is added at run time */
```

Comment:
@reddymur can you please point me to the code where we add this suffix at runtime?

Reply:
sound/soc/sof/intel/hda.c => @line 1518

Reply:
thanks!